docs/en/data_types/special_data_types/interval.md

```diff
@@ -0,0 +1,74 @@
+# Interval {#data-type-interval}
+
+Represents time and date intervals. The resulting type of the `INTERVAL` operator.
+
+!!!warning "Warning"
+    You can't use the `Interval` data type for storing values in tables.
+
+Structure:
+
+- Unsigned integer value of interval.
+- Type of interval.
+
+Supported interval types:
+
+- SECOND
+- MINUTE
+- HOUR
+- DAY
+- WEEK
+- MONTH
+- QUARTER
+- YEAR
+
+For each interval type, there is the separated data type. For example, the DAY interval is expressed as the `IntervalDay` data type:
```

Comment:
I think we'd better explicitly explain that this as a set of similarly behaving data types from the very beginning